add support for OneRingTranslator (local translation REST server)

janvarev opened the pull request:
I've added support to use as translation provider local server OneRingTranslator: https://github.com/janvarev/OneRingTranslator
It supports LibreTranslate, offline translation by FB NLLB (local installation if you need it) and some others.

HIllya51 commented:
This seems to be just a forwarding for other translation APIs, not a real translation API.
Actually the program has reserved settings that allow users to use their own translation interface for translation in Offline translation->Custom translation.
If you still want to add such a very complex interface and think it is an offline translation, you can do this:
1. Add oneringtranslator to "fanyi_offline" in
 https://github.com/HIllya51/LunaTranslator/blob/main/LunaTranslator/files/defaultconfig/static_data.json
2. Add some descriptions and settings to help users understand how to use it in  https://github.com/HIllya51/LunaTranslator/blob/main/LunaTranslator/files/defaultconfig/translatorsetting.json

janvarev commented:
Thanks for response!
I've added data (as I understand, sorry, if it's not super correct), and refactored my Readme.md here: https://github.com/janvarev/OneRingTranslator to make it more clearer "how to setup offline translation"

janvarev commented:
@HIllya51 Oh, I've skipped this! Fixed now.
